### PR TITLE
revise content based on the feedback from Ben

### PR DESCRIPTION
**Pull request by geek-yang**
Revise content of storyboard Atlas based on the feedback from Ben.

### CODE REVIEW

**Comment by geek-yang** [comment]
@Peter9192 I put the revised figures into a new .pptx file marked as "atlas_ben_booth_v2.pptx", which you can find in the EUCP folder in onedrive.

File: content/atlas/2_methods.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-headline: Methods
+headline: Underlying understanding
 ---
 
-## Systematic assessment of potential approaches in EUCP to aid decision adoption
+## EUCP has led underlying work to understand user value in these approaches
 
-In EUCP, we evaluated several methods to constrain climate projections, focusing
-on the following questions:
+The work within EUCP has been on understanding different potential quantitative approaches to using historical validation of current projections.  We provided the first common evaluation of their projected changes, across a range of currently available methodologies.
```
Comment (Peter9192) [inline content/atlas/2_methods.md]:
```suggestion
The work within EUCP has been on understanding different potential quantitative approaches to using historical validation of current projections. We provided the first common evaluation of their projected changes, across a range of currently available methodologies. 
```

File: content/atlas/2_methods.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-## Systematic assessment of potential approaches in EUCP to aid decision adoption
+## EUCP has led underlying work to understand user value in these approaches
 
-In EUCP, we evaluated several methods to constrain climate projections, focusing
-on the following questions:
+The work within EUCP has been on understanding different potential quantitative approaches to using historical validation of current projections.  We provided the first common evaluation of their projected changes, across a range of currently available methodologies. 
 
-* Do approaches lead to different projected changes?
-* Does the use of observations lead to more reliable climate projections?
+We have assessed the reliability of these approaches using blind out-of-sample evaluation, using new CMIP6 simulations as a proxy for real world historical and future responses.  This has enabled us to look at the skill of each method across a range of metrics, which provides an objective assessment of a particular methodological approach.
```
Comment (Peter9192) [inline content/atlas/2_methods.md]:
```suggestion
We have assessed the reliability of these approaches using blind out-of-sample evaluation, using new CMIP6 simulations as a proxy for real world historical and future responses. This has enabled us to look at the skill of each method across a range of metrics, which provides an objective assessment of a particular methodological approach.
```

**Review by Peter9192 — APPROVED** [review]
Looks much better now. One final comment is that the captions on the first panel are not clearly recognizable as captions when you first look at them. Perhaps make them smaller, italic, and/or put "Above: ..." and "Left: ..." labels in front of them?

**Comment by Peter9192** [comment]
Oh and also remove the "in preparation" label please (in two places)

File: content/atlas/2_methods.md (hunk 1 above)
Comment (geek-yang) [reply content/atlas/2_methods.md]: Corrected!

File: content/atlas/2_methods.md (hunk 2 above)
Comment (geek-yang) [reply content/atlas/2_methods.md]: Corrected!

**Pull request merged**